packages/react-component-manifests/src/manifest.config.js

```diff
@@ -169,5 +169,29 @@ module.exports = {
       modulePath: "./src/manifests/Cascader/Cascader.tsx",
       exportedVarName: "Cascader",
     },
+    Tabs: {
+      modulePath: "./src/manifests/Tabs/Tabs.tsx",
+      exportedVarName: "Tabs",
+    },
+    Textarea: {
+      modulePath: "./src/manifests/Textarea/Textarea.tsx",
+      exportedVarName: "Textarea",
+    },
+    Search: {
+      modulePath: "./src/manifests/Search/Search.tsx",
+      exportedVarName: "Search",
+    },
+    Select: {
+      modulePath: "./src/manifests/Select/Select.tsx",
+      exportedVarName: "Select",
+    },
+    Typography:{
```

Comment:
We don't need to update the `manifest.config.js` file anymore. I will actually remove this exported object from `manifest.config.js` file.